android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -164,6 +166,10 @@ protected void handleOpen(KrollDict options)
 	            topActivity.startActivity(intent);
 	        }
 	    }
+	    
+	    if (options.containsKey(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE)) {
+	        setPropertyAndFire(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE, options.getBoolean(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE));
```

Comment:
So where is the property used on the native side? I only see the proxy-methods so far. WiP?